modules/standard/Math.chpl

```diff
@@ -1056,6 +1056,21 @@ module Math {
     extern proc truncf(x: real(32)): real(32);
     return truncf(x);
   }
+  
+  /* Returns the greatest common divisor of the integer argument `arg1` and
+     `arg2`. */
```

Comment:
The documentation refers to `arg1` and `arg2` but these arguments are called `a` and `b`. This is important because you can pass arguments by name in Chapel (e.g. `gcd(b=24, a=8)`.

Reply:
Yeah.. minor mistake sorry sir for bothering again . I will update it.